server/methods/core/shop.js

```diff
@@ -121,16 +127,16 @@ Meteor.methods({
     const newShop = Collections.Shops.findOne({ _id: newShopId });
 
     // we should have created new shop, or errored
-    Logger.info("Created shop: ", shop._id);
+    Logger.info("Created shop: ", newShopId);
```

Comment:
better to use the _id returned from Shops.insert just in case the passed value is ignored (and auto generated... which... it really should be, right?)